tests/test_federation/utils.py

```diff
@@ -131,7 +132,12 @@ async def async_direct_link(ids):
     Node('CartItem', [
         Field('id', Integer, cart_item_resolver),
         Field('cart_id', Integer, cart_item_resolver),
-        Field('name', String, cart_item_resolver),
+        Field(
+            'name',
+            String,
+            cart_item_resolver,
+            directives=[Deprecated('do not use')]
```

Comment:
`hiku.graph.Link` is also a "field" in GraphQL terms, so it also can have field-level directives

Reply:
Thanks, will implement directives for links as well

Reply:
done